modules/tide_media/tide_media.install

```diff
@@ -81,3 +81,35 @@ function tide_media_update_10005() {
   $config->set('track_enabled_target_entity_types', $result);
   $config->save();
 }
+
+/**
+ * Adds secure file widget to entity browser.
+ */
+function tide_media_update_10006() {
```

Comment:
this should be in the` tide_media_secure_files` module instead of adding it here. Cause this should only exists if this module is enabled. So move this in the related module as a update hook and also put the same function in the install hook, so that it gets into the project when `tide_media_secure_files` is installed first time.